web/src/pages/dashboard/state-dashboard/ApdList.js

```diff
@@ -80,18 +76,17 @@ const ApdList = ({
                     {state ? state.name : ''} APDs
                   </h2>
                   {canCreateApd && (
-                    <Button
-                      variation="primary"
-                      className="ds-u-float--right"
-                      onClick={createNew}
+                    <Link
```

Comment:
I think you could have done a routing change within the `createNew` of the `onClick` in the button. This seems fine too though

Reply:
I think leaving it as a button might be better a11y-wise, but I'm not positive on that.

Reply:
A button is usually to enact a function (i.e. - creating an apd) whereas a link directs a user to another page?